Techome/Techome/Views/TotalList/TotalListView.swift

```diff
@@ -0,0 +1,260 @@
+                        SideEffectRecordRow()
+                    }
+                    else {
+                        
+                    }
+                    
+                    //마지막 데이터 다음 divider 없애기
+                    if (idx != tmpDataTotalListArr.count - 1) {
+                        Divider()
+                            .padding(.horizontal, TotalListLayoutValue.Paddings.dividerHorizontalPadding)
+                    }
+                    
+                }
+
+            }
+            .padding(.vertical, 3)
+            .background(Color.white)
+            .clipShape(RoundedRectangle(cornerRadius: 7))
+            .shadow(color: Color.primaryShadowGray, radius: 7, x: 0, y: 0)
+            
+        }
+    }
+    
+}
+
+
+
+//부작용 데이터
+
+//부작용 시간 + 부작용 정보
+struct SideEffectRecordRow: View {
+    
+    var body: some View {
+        
+        VStack(alignment: .leading, spacing: 0) {
+            
+            HStack(spacing: 0){
+            
+                Text("09:30")
+                    .font(.caption)
+                    .foregroundColor(.secondaryTextGray)
+                Image(systemName: "exclamationmark.circle")
+                    .resizable()
+                    .frame(width: 15, height: 15)
+                    .foregroundColor(.customRed)
+                    .padding(.leading, 5)
+            }
+            .padding(.leading, TotalListLayoutValue.Paddings.sideEffectRecordRowHorizontalPadding)
+            
+            SideEffectRecordsByDayList()
+            
+        }
+        .padding(.top, TotalListLayoutValue.Paddings.sideEffectRecordRowVerticalPadding)
+    }
+    
+}
+
+//============================================================================
+//trend에서 사용하는 컴포넌트 공유 예정
+
+
+//trend 의 sideEffectRecordsByDay 와 동일
+//부작용 데이터
+
+struct SideEffectRecordsByDayList: View {
+    var body: some View {
+        VStack(alignment: .center, spacing: TotalListLayoutValue.Paddings.sideEffectRecordCellVerticalPadding) {
+            ForEach(0..<2) { sideEffectRowIndex in
+                HStack(alignment: .center, spacing: TotalListLayoutValue.Paddings.sideEffectRecordCellHorizontalPadding) {
+                    //TODO: 임시 데이터 수
+                    ForEach(0..<5) { sideEffectItemIndex in
+                        SideEffectRecordItemList()
+                    }
+                }
+            }
+        }
+        .padding(.vertical, TotalListLayoutValue.Paddings.sideEffectRecordCellVerticalPadding)
+        .frame(width: TotalListLayoutValue.Sizes.cardWidth)
+    }
+}
+
+//trend 의 SideEffectRecordItem 과 동일
+struct SideEffectRecordItemList: View {
+    var body: some View {
+        VStack(spacing: 0) {
+            Image("esophagitis")
+            Text("식도염")
+                .font(.caption)
+        }
+        .frame(width: TotalListLayoutValue.Sizes.sideEffectRecordCellFixedWidth)
+    }
+}
+
+
+
+//trend 의 CaffeineRecordCell 에 padding (.vertical)로 제한하는 코드 추가 & Divider 삭제, 이외 동일
```

Comment:
```suggestion
// TODO: trend 의 CaffeineRecordCell 에 padding (.vertical)로 제한하는 코드 추가 & Divider 삭제, 이외 동일
```

남은 작업과, 메모를 위한 주석의 구분이 되면 좋을 것 같아요 ~

Reply:
네, 알겠습니다🙂